New gs english (#2497)
tarantool/doc: doc/getting_started/change_schema_dynamically.rst

.. _getting_started-schema_changing:

=================================================================================
Updating the data schema
=================================================================================

When working with data, it is sometimes necessary to change the original data schema.

> In the previous sections, we described the data schema across a cluster in YAML format.

Reviewer: ```suggestion
In the previous sections, we described a cluster-wide data schema in the YAML format.
```
if I understand it correctly

The ``ddl`` module is responsible for applying the schema on the cluster. This module does not allow

> modify the schema after applying it.

Reviewer: ```suggestion
to modify the schema after applying it.
```

> The easiest way to change it is to delete the database snapshots and create
> scheme from scratch. Of course, this is only acceptable during development and debugging.

Reviewer: ```suggestion
The easiest way to change it is to delete the database snapshots and create
a schema from scratch. Of course, this is only acceptable during development and debugging.
```
In American English, "scheme" is an evil plan. "Schema" is database-related.
